src/rebuild/scan.c

```diff
@@ -706,6 +708,9 @@ rebuild_obj_scan_cb(daos_handle_t ch, vos_iter_entry_t *ent,
 	}
 
 	grp_size = daos_oclass_grp_size(oc_attr);
+	grp_nr   = daos_obj_id2grp_nr(oid.id_pub);
+	/* appropriate yield based on shard number */
+	arg->yield_cnt -= roundup(grp_size * grp_nr, 128) / 128;
```

Comment:
is the use of the constant 128 here meant to be identical to SCAN_YIELD_CNT value? If so, maybe use that macro in this code too?

Reply:
the 128 here is not identical to SCAN_YIELD_CNT macro value, just a estimation of the layout calculation overhead.